group_project_v2/stage.py

```diff
@@ -714,7 +715,7 @@ def other_submission_links(self, request, suffix=''):  # pylint: disable=unused-
     @outsider_disallowed_protected_handler
     @key_error_protected_handler
     def load_other_group_feedback(self, request, suffix=''):  # pylint: disable=unused-argument
-        group_id = request.GET["group_id"]
+        group_id = int(request.GET["group_id"])
```

Comment:
@Kelketek this is not as much protected as other two, so theoretically it could throw conversion error. I'll update the PR later.